parcels/particle.py

```diff
@@ -224,12 +234,12 @@ def Kernel(self, pyfunc):
         return Kernel(self.grid, self.ptype, pyfunc)
 
     def ParticleFile(self, *args, **kwargs):
-        return ParticleFile(*args, particleset=self, **kwargs)
+        return ParticleFile(*args, particleset=self, user_vars=self.ptype.user_vars, **kwargs)
 
 
 class ParticleFile(object):
 
-    def __init__(self, name, particleset, initial_dump=True):
+    def __init__(self, name, particleset, initial_dump=True, user_vars=None):
```

Comment:
The `particleset` has an associated `ParticleType ptype` attribute, which contains the `user_vars`. We could just infer it from there instead of adding to the constructor, no?